### PR TITLE
Update Apparel_RS_CE.xml

### PR DESCRIPTION
**Pull request by JustCrulien**
A very novice attempt at patching Rimsenal core apparel, I only saw Tostov was making one after I finished so I guess this was mostly just practice to make other mods compatible.

## Additions
-Added compatibility with new Rimsenal core armors, values in line with old armor

-changed the way cost list is patched (overwrites the whole list tho so if the author added a new ingredient to the cost list that was relevant like the <MBPSKit> it would be overwritten.
The other conditional way I have below is better I think.

-Made korps (artillery armor) use 60mm mortar just wanted CE rated ammunition on it, it looks cool firing and I didn't check the weights but a few 60mm shells might be more feasible to carry around than 80x I_Alloy for every shot(not to say that it wasn't a good idea, its very fitting). (unless its meant to be a one shot per fight type of weapon but the vanilla mod had shells as ammo,  the emp launcher armor used 25 steel in vanilla because losing a whole emp grenade weapon in vanilla would hurt)

-Gave corps a 160 carry weight to be able to carry a good loadout(mass 110 so only 50 extra minus the helmet weight of 10 for 40)

-Changed korps movement penalty from -1.5 to -1 this is mostly for me personally cause the tree people race has a movement speed of only 2 and it just would be weird to have them basically crippled + anyone that got hurt would walk so slow

-Added the extra cost of medic, pioneer and nomad armor to the MBPSKit, just though this is what the author wanted since in the description it says that you can upgrade it quickly from a kit, so like if you bought it from traders you could upgrade it without being a Tec civ.

## Changes

-Replaced drop suit max charges from 3 to 6 cause locust armor was better option in my opinion
added functionality
## Reasoning
 -Just did what I though was intended and what would fit well

## Alternatives

-Was thinking of making a conditional call looking for "commando mortars" and if that mod is active, change the stats on korps cause he becomes redundant, if any pawn wearing power amour can carry a mortar launcher what's the point of korps losing all those others stats.

-Added new method for changing cost values Example in use on Korps and MSBSkit.
-If you used this template to change the Cost list I don't think the original mod removing values would break anything.
-If its actually a good method I can update the rest of the entries.
                       
                       <li Class="PatchOperationConditional">
				<xpath>/Defs/thingDef[defName="ArmorX"]/costList/Plasteel</xpath>
			<match Class="PatchOperationSequence">
			<operations>

					<li Class="PatchOperationRemove">
						<xpath>/Defs/thingDef[defName="ArmorX"]/costList/Plasteel</xpath>
					</li>
				</operations>
				</match>
			</li>

			<li Class="PatchOperationConditional">
				<xpath>/Defs/thingDef[defName="ArmorX"]/costList/Steel</xpath>
			<match Class="PatchOperationSequence">
			<operations>

					<li Class="PatchOperationRemove">
						<xpath>/Defs/thingDef[defName="ArmorX"]/costList/Steel</xpath>
					</li>
				</operations>
				</match>
			</li>	

			<li Class="PatchOperationConditional">
				<xpath>/Defs/thingDef[defName="ArmorX"]/costList/Uranium</xpath>
				<match Class="PatchOperationSequence">
				<operations>

					<li Class="PatchOperationRemove">
						<xpath>/Defs/thingDef[defName="ArmorX"]/costList/Uranium</xpath>
					</li>	
				</operations>
				</match>
			</li>

			<li Class="PatchOperationConditional">
				<xpath>/Defs/thingDef[defName="ArmorX"]/costList/DevilstrandCloth</xpath>
				<match Class="PatchOperationSequence">
				<operations>

					<li Class="PatchOperationRemove">
						<xpath>/Defs/thingDef[defName="ArmorX"]/costList/DevilstrandCloth</xpath>
					</li>
				</operations>
				</match>
			</li>

                        <li Class="PatchOperationAdd">
				<xpath>Defs/thingDef[defName="ArmorX"]/costList</xpath>
				<value>
					<Plasteel>x</Plasteel>
					<DevilstrandCloth>x</DevilstrandCloth>
					<Steel>x</Steel>	
				</value>

## Testing

Check tests you have performed:
- [no] Compiles without warnings "korps can't be set to have forced miss radius, forced miss radius is for explosive weapons"  so I just commented out that line. no other error
- [y] Game runs without errors
- [ ] (For compatibility patches) ...with and without patched mod loaded
- [y] Playtested a colony (specify how long) (2 hour full mod list no new errors besides forced miss radius)

### CODE REVIEW

File: Patches/Rimsenal Collection/Federation/ThingDefs_Misc/Ammo_Fed.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
 		</graphicData>		
 		<projectile Class="CombatExtended.ProjectilePropertiesCE">
 			<damageDef>Plasma</damageDef>
-			<speed>50</speed>
+			<speed>0</speed>
```
Comment (Alicecomma) [inline Patches/Rimsenal Collection/Federation/ThingDefs_Misc/Ammo_Fed.xml:50]: Won't this throw errors?

Comment (N7Huntsman) [reply Patches/Rimsenal Collection/Federation/ThingDefs_Misc/Ammo_Fed.xml:50]: Not if this is a projectile for a mortar-like weapon.

**Review by N7Huntsman — APPROVED** [review]
What's here seems to be fine, but there remains some other patched RS content that needs work done on it.

**Comment by JustCrulien** [comment]
sure thing! I stopped doing rimsenal cause Tostov was doing it, but if there's anything you want done I can pause what I'm doing and do that. Right now I'm trying to patch "[Kir]Extra Mechanoid Units and Equipment" by looking at the VFE Mechs patch.

**Pull request merged**